tests/models/arith/arith10.optimize.expected

```diff
@@ -2,5 +2,9 @@
 unknown
 (
   (define-fun x () Real 0.0)
-  (define-fun y () Real 10.0)
+  (define-fun y () Real 0.0)
 )
+(objectives 
```

Comment:
As y cannot be optimized, I believe we should warn the user when he invokes (get-objectives)